src/Application.cpp

```diff
@@ -669,6 +729,7 @@ bool Application::loadGame(const std::string& path)
   }
 
   _gamePath = path;
+  _gameFileName = unzippedFileName;
```

Comment:
`_gameFileName` contains the filename "foo.nes" regardless of whether it was loaded directly or from inside the zip. This allows the sram file to be named `foo.nes.sram` instead of `foo.zip.sram`.

`_gamePath` contains the full path to the original file/zip.